first-time.sql

```diff
@@ -0,0 +1,298 @@
+  PRIMARY KEY (`log_id`),
+  KEY `WIKI` (`wiki`),
+  KEY `LOCALE` (`locale`),
+  KEY `LOGTYPE` (`log_type`),
+  KEY `LOGACTION` (`log_action`),
+  KEY `LOGOBJECT` (`log_object`),
+  KEY `LOGUSER` (`log_user`),
+  KEY `LOGTIMESTAMP` (`log_timestamp`),
+  KEY `LOGSPECIFIC` (`log_type`,`log_action`,`log_object`,`log_user`)
+) ENGINE=InnoDB AUTO_INCREMENT=5 DEFAULT CHARSET=utf8mb4;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `externallinks_userlog`
+--
+
+LOCK TABLES `externallinks_userlog` WRITE;
+/*!40000 ALTER TABLE `externallinks_userlog` DISABLE KEYS */;
+INSERT INTO `externallinks_userlog` VALUES (1,'testwiki','testwiki','tos','accept',0,'',NULL,NULL,1,'2021-06-16 23:40:57',''),(2,'testwiki','testwiki','wikiconfig','change',0,'',NULL,NULL,1,'2021-06-17 00:05:39',''),(3,'testwiki','testwiki','wikiconfig','change',0,'',NULL,NULL,1,'2021-06-17 00:06:13',''),(4,'global','testwiki','usergroups','define',0,'',NULL,NULL,1,'2021-06-17 00:07:43','');
+/*!40000 ALTER TABLE `externallinks_userlog` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `externallinks_userpreferences`
+--
+
+DROP TABLE IF EXISTS `externallinks_userpreferences`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `externallinks_userpreferences` (
+  `user_link_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `user_email` blob DEFAULT NULL,
+  `user_email_confirmed` tinyint(1) NOT NULL DEFAULT 0,
+  `user_email_confirm_hash` varchar(32) DEFAULT NULL,
+  `user_email_fpreport` tinyint(1) NOT NULL DEFAULT 0,
+  `user_email_blockstatus` tinyint(1) NOT NULL DEFAULT 1,
+  `user_email_permissions` tinyint(1) NOT NULL DEFAULT 1,
+  `user_email_fpreportstatusfixed` tinyint(1) NOT NULL DEFAULT 1,
+  `user_email_fpreportstatusdeclined` tinyint(1) NOT NULL DEFAULT 1,
+  `user_email_fpreportstatusopened` tinyint(1) NOT NULL DEFAULT 1,
+  `user_email_bqstatuscomplete` tinyint(1) NOT NULL DEFAULT 1,
+  `user_email_bqstatuskilled` tinyint(1) NOT NULL DEFAULT 1,
+  `user_email_bqstatussuspended` tinyint(1) NOT NULL DEFAULT 1,
+  `user_email_bqstatusresume` tinyint(1) NOT NULL DEFAULT 1,
+  `user_new_tab_one_tab` tinyint(1) NOT NULL DEFAULT 1,
+  `user_allow_analytics` tinyint(1) NOT NULL DEFAULT 0,
+  `user_default_wiki` varchar(45) DEFAULT NULL,
+  `user_default_language` varchar(45) DEFAULT NULL,
+  `user_default_theme` varchar(45) DEFAULT NULL,
+  PRIMARY KEY (`user_link_id`),
+  KEY `HASEMAIL` (`user_email_confirmed`)
+) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8mb4;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `externallinks_userpreferences`
+--
+
+LOCK TABLES `externallinks_userpreferences` WRITE;
+/*!40000 ALTER TABLE `externallinks_userpreferences` DISABLE KEYS */;
+INSERT INTO `externallinks_userpreferences` VALUES (1,NULL,0,NULL,0,1,1,1,1,1,1,1,1,1,1,0,NULL,NULL,NULL);
```

Comment:
You may want to drop this line.  It's not attached to a user ID.  However, if this wasn't an issue during testing, you can leave it.

Reply:
Thanks, I'll keep it to avoid yet another unneeded screen for dev setup.